camel/models/nemotron_model.py

```diff
@@ -46,9 +46,9 @@ def __init__(
         api_key: Optional[str] = None,
         url: Optional[str] = None,
     ) -> None:
+        url = url or os.environ.get("NVIDIA_API_BASE_URL")
```

Comment:
Should we also provide the default `NVIDIA_API_BASE_URL` and remove the raise `ValueError`? Just like `SambaModel`.